### PR TITLE
chore: removes read usage of kumaApi

### PR DESCRIPTION
**Pull request by johncowen**
Generally our onboarding area is pretty frozen, but it still has older code that we have to continue supporting unless we get rid of it (in this case `kumaApi` usage vs DataSource/DataLoader).

I had a bit of time whilst waiting on other PRs so, this PR switches the remaining areas of onboarding to use DataSource/DataLoader, and in doing so:

1. Removes all the older imperative code from onboarding and replaces it with DataSource
2. Changes some places where we were looking for a count of at least one zone/dataplane/zone-ingress and instead looks for at least a count of one _and_ one is online.
3. Uses the newer `_overview` endpoints for gathering data
4. Removes the last occurrences of us using `useKumaApi()` / `kumaApi` for reading data, which means we are closer to being able to use an OpenAPI spec auto-generated typescript http client.
5. I noticed that the URL for the documentation link was wrong in multizone onboarding, so I corrected that with a little more i18n markdown-ing while I was here. This then led to being unable to use `v-html` and `class` on the same element due to a clash with eslint rules (`class` has to be above `v-html` which means you can't disable the `v-html` on the next line because an eslint fix that reorders `class` and `v-html` means its no longer on the next line). Due to this I disabled the no-v-html rule entirely. Thinking being, we need to/will need to use this a lot. That being said, `v-html` should only be used for i18n purposes.

I might add some inlines as this PR snowballed into something bigger than I originally imagined.

---

An important note here. Ideally our DataSource URIs would be split into using plural-name/singular-name i.e. `/zone-cps` and `/zone-cp/:name`. This means we can safely add 'actions' on collections of things without worrying about clashing with the `:name` part of a singular. 

For example, in the case of this PR, I very easily clashed with `/zone-cps/online` with `/zone-cps/:name` without realising. If we used `/zone-cp/:name` instead (i.e. the singular for the singular request) this wouldn't be an issue. Also importantly, if we ever need to perform an 'action' request on a singular item the URI would be `/zone-cp/:name/:action` meaning we again avoid the clash.

To temporarily work around this issue in this PR I've used the 'DNS trick' `/zone-cps/~online` seeing as we can't use the `~` character in the name of a zone-cp and ordered the source to come before `/zone-cps/:name` therefore avoiding a potential clash. But we should change all our source URIs to have plural/singular forms.

### CODE REVIEW

File: .eslintrc.cjs
Comment (johncowen) [inline .eslintrc.cjs:99]: If there is a suggestion for an alternative to doing this without it being annoying it would be much appreciated.

File: features/onboarding/add-new-services-code/Index.feature
Comment (johncowen) [inline features/onboarding/add-new-services-code/Index.feature:23]: We now check `_overview` for counts of things, I think this is fine, but let me know if not.

File: features/onboarding/multi-zone/Index.feature
Comment (johncowen) [inline features/onboarding/multi-zone/Index.feature:24]: Again we use `/_overview` here now

File: src/app/onboarding/components/OnboardingAlert.vue
Comment (johncowen) [inline src/app/onboarding/components/OnboardingAlert.vue:13]: I just removed the comment from here seeing as we disable it globally.

File: src/app/onboarding/locales/en-us/index.yaml
Comment (johncowen) [inline src/app/onboarding/locales/en-us/index.yaml:25]: Corrected the docs URL, and i18n'ed the rest while I was here.

File: src/app/onboarding/views/OnboardingAddNewServicesCodeView.vue
Comment (johncowen) [inline src/app/onboarding/views/OnboardingAddNewServicesCodeView.vue:99]: This is mostly a re-nest due to the DataSource

File: src/app/onboarding/views/OnboardingDataplanesView.vue
Comment (johncowen) [inline src/app/onboarding/views/OnboardingDataplanesView.vue:70]: I noticed that `KTable::fetcher` isn't reactive, i.e. if i change the value of `:fetcher` `KTable` will still use the initial one, which I guess is probably a bug. Not totally sure i might file an issue. Wdyt? Should this be considered a bug?

---

I left in the function factory incase it ever gets fixed

**Review by johncowen — COMMENTED** [review]
Inlines:

**Comment by slonka** [comment]
@johncowen do you need any help to move this forward? it was stuck for 3 weeks

**Comment by johncowen** [comment]
Yeah it would be good to get a review on it. I think most of the change is just re-using patterns from what we have elsewhere so there shouldn't be anything to controversial.

I think maybe the only places I was specifically hoping for a second pair of eyes was these things from the PR desc:

> 2. Changes some places where we were looking for a count of at least one zone/dataplane/zone-ingress and instead looks for at least a count of one and one is online.
> 3. Uses the newer _overview endpoints for gathering data

Just an additional note incase, there is nothing urgent here and nothing here is needed for 2.7, but in the same breath it would be good to get this merged, otherwise we may as well close it (which would be a shame as it is valuable seeing as we want to get to a OpenAPI generated HTTP client)

If you want to take a look yourself lemme know if I can help further with anything

**Review by kleinfreund — APPROVED** [review]

**Comment by johncowen** [comment]
Rebased this one also, will hold off merge for a little while incase, ty!

**Pull request merged**
